version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.1.7-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.2.0"
```

Comment:
releasing should be fairly simple (https://github.com/xebia/cakemix/blob/master/CONTRIBUTING.md#releasing)